src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -151,22 +164,83 @@ export const useSwapper = () => {
         throw new Error('Missing buyAccountBip44Params')
       if (!sellAccountMetadata) throw new Error('Missing sellAccountMetadata')
 
-      const trade = await getTradeForWallet({
+      const activeSwapper = activeSwapperWithMetadata?.swapper
```

Comment:
all of the following green was lifted out of `useSwapperStore` `selectGetTradeForWallet` to prevent dependency cycle